src/libraries/System.IO.FileSystem/tests/Base/AllGetSetAttributes.cs

```diff
@@ -59,7 +59,15 @@ public void SymLinksReflectSymLinkAttributes()
             try
             {
                 Assert.Equal(FileAttributes.ReadOnly, FileAttributes.ReadOnly & GetAttributes(path));
-                Assert.NotEqual(FileAttributes.ReadOnly, FileAttributes.ReadOnly & GetAttributes(linkPath));
+                if (OperatingSystem.IsWindows())
+                {
+                    Assert.NotEqual(FileAttributes.ReadOnly, FileAttributes.ReadOnly & GetAttributes(linkPath));   
+                }
+                else
+                {
+                    // On Unix, Get/SetAttributes FileAttributes.ReadOnly operates on the target of the link.
```

Comment:
This in a change in behavior:

On Unix, `SetAttributes` (for `FileAttributes.ReadOnly`) was already operating on the link target.
With the changes in this PR, `SetAttributes`  also operates on the target.

Reply:
Do you think it needs a breaking change doc?

Reply:
I don't find it necessary. I consider this a bugfix more than a breaking change.